Neo4j.Driver/Neo4j.Driver/Internal/Auth/RotatingClientCertificateProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [https://neo4j.com]
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Neo4j.Driver.Internal.Auth;
+
+internal class RotatingClientCertificateProvider : IRotatingClientCertificateProvider
+{
+    private readonly object _lock = new();
```

Comment:
I think use a RWLock here or we create sequential read access

Reply:
done